kyuubi-ctl/pom.xml

```diff
@@ -32,6 +32,11 @@
     <packaging>jar</packaging>
     <url>https://kyuubi.apache.org/</url>
 
+    <properties>
+        <!-- only used in kyuubi-ctl module -->
+        <snakeyaml.version>1.30</snakeyaml.version>
```

Comment:
Let manage it as other deps, if you check the `dependencyList`, it's already be there, as a transitive dep of `jackson-dataformat-yaml`